translations/messages-en.properties

```diff
@@ -906,4 +906,7 @@ forms.json = JSON
 upload.xform.xml.help = The xform definition in XML.
 upload.xform.meta.help = The meta file in JSON format including the icon, title, context, and so on.
 password.weak = The password is too easy to guess. Include a range of types of characters to increase the score.
-password.length.minimum = The password must be at least {{minimum}} characters long.
\ No newline at end of file
+password.length.minimum = The password must be at least {{minimum}} characters long.
```

Comment:
If you configure your editor to include linebreaks at the end of files, git won't consider the previous-final line of files to have changed when you append more lines later.